config/pipeline.yaml

```diff
@@ -813,6 +824,16 @@ jobs:
       tree:
       - 'android'
 
+  kbuild-gcc-12-x86-chrome:
+    <<: *kbuild-gcc-12-x86-job
+    params:
+      <<: *kbuild-gcc-12-x86-params
+      fragments:
+        - x86-board
```

Comment:
Would having the `lab-setup` fragment be a problem? If the answer is no, then there's no need for this job, it will run on `kbuild-gcc-12-x86` and `kbuild-gcc-12-x86-chromeos` (which has 2 more options, but nothing really significant).

Similarly, those build jobs would run for arm64 as well with `kbuild-gcc-12-arm64-chromeos`.

Overall, IMHO adding new jobs is unnecessary here, so the only changes needed would be:
* add the tree and build configs
* allow this tree for the arm build as you did above

Reply:
Removed kbuild-gcc-12-x86-chromeos. kbuild-gcc-12-arm64-chromeos cannot be removed as there is only kbuild-gcc-12-arm64-kcidebug job which could be used if it didn't have kcidebug fragment.

Reply:
Edited above comment.

I'm confused. Let me share what I've understood.
`kbuild-gcc-12-arm64-chromeos` is an anchor in jobs-chromeos.yaml. `kbuild-gcc-12-arm64-chromebook` is the job using the anchor `kbuild-gcc-12-arm64-chromeos` which builds all trees other than android. Thus should I just remove `kbuild-gcc-12-arm64-chrome` from `pipeline.yaml`?

Reply:
@a-wai what do you think?